### PR TITLE
STORM-3064: PartitionedTridentSpoutExecutor should use getPartitionsF…

### PR DESCRIPTION
**Pull request by milantracy**
…orTask

### CODE REVIEW

File: storm-client/src/jvm/org/apache/storm/trident/spout/IPartitionedTridentSpout.java [hunk 1]
```diff
@@ -36,7 +37,7 @@
     interface Coordinator<Partitions> {
         /**
          * Return the partitions currently in the source of data. The idea is is that if a new partition is added and a prior transaction is
-         * replayed, it doesn't emit tuples for the new partition because it knows what partitions were in that transaction.
+         * replayed, it doesn't emit tuples for the new partition because it knows what partitions were in that transaction.jobh
          */
```
Comment (danny0405) [inline storm-client/src/jvm/org/apache/storm/trident/spout/IPartitionedTridentSpout.java]: Sorry, what is the jobh mean?

Comment (srdo) [reply storm-client/src/jvm/org/apache/storm/trident/spout/IPartitionedTridentSpout.java]: Whoops, yes. Please remove this.

Comment (milantracy) [reply storm-client/src/jvm/org/apache/storm/trident/spout/IPartitionedTridentSpout.java]: thanks for comments folks, I remove typos here and add more comments

**Comment by srdo** [comment]
@milantracy Looks great. Please squash to one commit, and we can merge once the 24 hour waiting period is over.

**Comment by milantracy** [comment]
@srdo @danny0405 squashed, thanks for reviewing :)

**Pull request merged**